config/manager.go

```diff
@@ -174,16 +175,10 @@ func (f *FilesystemConfigManager) load(c *Config) error {
 
 	var data []byte
 
-	if f.fsHandle.FileExists(f.location) {
-		// #nosec G304 -- no input comes from the user
-		data, err = f.fsHandle.ReadFile(f.location)
-		if err != nil {
-			return err
-		}
-	} else {
-		if err := f.fsHandle.CreateFile(f.location, internal.PermUserRW); err != nil {
```

Comment:
Why this case is removed?

Reply:
It is deleted because:
* there is a check on line 164 if the file exist, and when it doesn't it returns immediately so this code will never execute
* if the code does executes, then there are other problems because this creates an empty file and decryption will always fail for empty files